packages/training/deploy/docker/Dockerfile.base

```
@@ -0,0 +1,79 @@
+#
+# ============================================================================
+FROM vllm/vllm-openai:v0.14.0
+
+# Note: vllm/vllm-openai uses Python 3.12 and includes:
+# - PyTorch with CUDA
+# - vLLM with FlashInfer backend
+# - No flash-attn by default (uses FlashInfer instead - actually better for inference!)
+
+# Install system dependencies
+USER root
+RUN apt-get update && apt-get install -y \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)